### DISCUSSION TITLE
Can we decrease or even remove the delay during the change of themes

### DISCUSSION THREAD

**Discussion by Aman1131145** [monkeytypegame/monkeytype#2823] [Ideas] [answered]
There is a delay while changing the themes I encounter it every time I start new test and the theme changes.
Sometimes the delay is so much that even if I start the test the theme won't apply and that is very annoying.

**Comment by Miodec** [comment]
How fast is your connection? The themes are quite small so the delay should be minimal. How does the website look during the transition? Does it have no style until the new theme is loaded? Or does it keep the style of the previous theme?

**Reply by Aman1131145** [reply]
my connection is good like 30 mbps or so even after that sometime i see 1 sec lag.

**Reply by Miodec** [reply]
Is the delay still there when changing themes manually?

**Comment by MaxACartwright** [comment, marked as answer]
If the delay *could* be removed, it would take a while for loading the website for the first time. Plus it would be bad for the cache.
